src/bindings/jna/libelektra/src/main/java/org/libelektra/KDB.java

```diff
@@ -67,52 +41,37 @@
 	 *                                  specialization of {@link KDBException}
 	 * @throws KeySetReleasedException  if {@code contract} has already been
 	 *                                  released
-	 * @throws IllegalArgumentException if {@code contract} or {@code errorKey} is
-	 *                                  {@code null}
+	 * @throws IllegalArgumentException if {@code contract} is {@code null}
+	 * @see #open()
 	 */
 	@Nonnull public static KDB open (KeySet contract) throws KDBException
```

Comment:
All parameters for all functions are deemed not to be allowed to be null because of the package level annotation `@javax.annotation.ParametersAreNonnullByDefault` (see `package-info.java`).
Therefore @Nonnull annotation is only necessary for return values.